versioned_docs/version-8.3/reference/supported-environments.md

```diff
@@ -30,11 +30,11 @@ Requirements for the components can be seen below:
 
 | Component   | Java version | Other requirements                                                                                                                                  |
 | ----------- | ------------ | --------------------------------------------------------------------------------------------------------------------------------------------------- |
-| Zeebe       | OpenJDK 17+  | Elasticsearch 8.8.x+                                                                                                                                |
-| Operate     | OpenJDK 17+  | Elasticsearch 8.8.x+                                                                                                                                |
-| Tasklist    | OpenJDK 17+  | Elasticsearch 8.8.x+                                                                                                                                |
+| Zeebe       | OpenJDK 17+  | Elasticsearch 8.8+                                                                                                                                  |
+| Operate     | OpenJDK 17+  | Elasticsearch 8.8+                                                                                                                                  |
+| Tasklist    | OpenJDK 17+  | Elasticsearch 8.8+                                                                                                                                  |
 | Identity    | OpenJDK 17+  | Keycloak 21.x, 22.x<br/>PostgreSQL 14.x, 15.x                                                                                                       |
-| Optimize    | OpenJDK 17+  | Elasticsearch 8.8.x+                                                                                                                                |
+| Optimize    | OpenJDK 17+  | Elasticsearch 8.8+                                                                                                                                  |
```

Comment:
@akeller - there is a separate matrix below for Optimize, do you think this should also be made consistent? The difference there is that for _some_ older versions, we didn't support all patches (see `7.16.2+` for example)

Reply:
@toco-cam removed this table from future docs - https://github.com/camunda/camunda-docs/pull/2670.

Reply:
This is complicated. Let's frame it like this - We are in the 8.3 docs, what version of Elasticsearch works will 8.3 + Optimize? Or more specifically, since Optimize can't operate in a vacuum, what ES version is required for 8.3? 

I'd love to get our supported environments to a point where it looks more like a unified platform experience, and the best way to get there is to be specific about what ES version works with the components together as a platform.

Can we follow up separately on the older Optimize versions? I think moving this Optimize table may help us here.

Reply:
Sure, makes sense. I would also argue that the old Optimize table is partly redundant. With versioned docs, we don't really need to list compatibility of older versions for a given docs page. You could also argue that we don't need to list compatibility for versions of Optimize no longer supported